PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj

```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="Hackney.Core.DynamoDb" Version="1.51.0" />
     <PackageReference Include="Hackney.Core.Testing.DynamoDb" Version="1.55.0" />
     <PackageReference Include="Hackney.Core.Testing.Shared" Version="1.54.0" />
-    <PackageReference Include="Hackney.Shared.PatchesAndAreas" Version="0.8.0" />
```

Comment:
Do we not want to keep this to numbered versions, or is this a provisional package?

Reply:
Once the [shared PR ](https://github.com/LBHackney-IT/patches-and-areas-shared/pull/4) gets approved I'll update this version number before merging the PR